packages/cms-lib/lib/config.js

```diff
@@ -449,6 +484,109 @@ const deleteEmptyConfigFile = () => {
   );
 };
 
+const getConfigVariablesFromEnv = () => {
+  const env = process.env;
+
+  return {
+    apiKey: env[HUBSPOT_API_KEY],
+    clientId: env[HUBSPOT_CLIENT_ID],
+    clientSecret: env[HUBSPOT_CLIENT_SECRET],
+    personalAccessKey: env[HUBSPOT_PERSONAL_ACCESS_KEY],
+    portalId: parseInt(env[HUBSPOT_PORTAL_ID], 10),
```

Comment:
Should the env variable just be `HUBSPOT_PORTAL`?

Reply:
I'd rather be verbose with the name so there is no question about it.